### PR TITLE
[IMP] l10n_hu: Update available banks list

### PR DESCRIPTION
**Pull request by kerteszgergo**
Lots of contact information has been changed and some banks has been sold their bisiness portfolio over the years. In some cases the telephone and email links were in the wrong order.

Description of the issue/feature this PR addresses:
Bank information is out of date.

Current behavior before PR:
Outdated bank information.

Desired behavior after PR is merged:
Have an up to date bank records.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kerteszgergo** [comment]
Updated the name. Now should be working.

**Comment by kerteszgergo** [comment]
I can't get rid of this name error even after I have changed the name.

**Comment by lordkrandel** [comment]
> I can't get rid of this name error even after I have changed the name.

It's the commit name. 👇🏻 
image

Commit name stays in `git log` and `git blame`.
The standard commit message format makes context information leap up to the eye! 😄 

image

Do a `git rebase -i 16.0` and then choose `reword`, then put the same as the PR title.

image

File: addons/l10n_hu/data/res.bank.csv [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-"GNBAHUHB","Gránit Bank Zrt.","GNBAHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 8.","info@granitbank.hu","+3640100777"
-"INCNHUHB","IC Bank Rt.","INCNHUHB","base.hu",1088,"Budapest","Rákóczi  út 1-3."," level@bancopopolare.hu ","+3640200515",
-"INGBHUHB","ING Bank N.V. Magyarországi Fióktelepe","INGBHUHB","base.hu",1068,"Budapest","Dózsa György  út 84.","communications.hu@ingbank.com","+362680140"
-"OKHBHUHB","K&H Bank Zrt.","OKHBHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 9."," bank@kh.hu ","+3613289000"
-"HBWEHUHB","MagNet Magyar Közösségi Bank Zrt.","HBWEHUHB","base.hu",1062,"Budapest","Andrássy utca 98.","info@magnetbank.hu","+3614288888"
+"GNBAHUHB","Gránit Bank Zrt.","GNBAHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 8.","info@granitbank.hu","+36 1 510 0527"
+"INCNHUHB","IC Bank Rt.","INCNHUHB","base.hu",1088,"Budapest","Rákóczi  út 1-3."," level@bancopopolare.hu ","+3640200515"
+"INGBHUHB","ING Bank N.V. Magyarországi Fióktelepe","INGBHUHB","base.hu",1068,"Budapest","Dózsa György út 84/b.","communications.hu@ingbank.com","+36 1 235 8700"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:13]: I see all the telephone numbers in the old file have no space inside.
Can you remove spaces from the new ones?

File: addons/l10n_hu/data/res.bank.csv [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+"OKHBHUHB","K&H Bank Zrt.","OKHBHUHB","base.hu",1095,"Budapest","Lechner Ödön fasor 9."," bank@kh.hu ","+36 1 328 9000"
+"HBWEHUHB","MagNet Magyar Közösségi Bank Zrt.","HBWEHUHB","base.hu",1062,"Budapest","Andrássy út 98.","info@magnetbank.hu","+36 1 428 8888"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:15]: See, here (and below this line too) the number didn't even change. It's just the spaces.

**Review by lordkrandel — CHANGES_REQUESTED** [review]
Some small cleanup needed

File: addons/l10n_hu/data/res.bank.csv [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
 "MANEHUHB","Magyar Nemzeti Bank","MANEHUHB","base.hu",1054,"Budapest","Szabadság tér 8-9.","info@mnb.hu","+3614282600"
-"MKKBHUHB","MKB Bank Zrt.","MKKBHUHB","base.hu",1056,"Budapest","Váci utca 38.","telebankar@mkb.hu","+3613278600"
-"OTPVHUHB","OTP Bank Nyrt.","OTPVHUHB","base.hu",1051,"Budapest","Nádor utca 16.",,"+3614735000"
-"UBRTHUHB","RAIFFEISEN Bank Zrt.","UBRTHUHB","base.hu",1054,"Budapest","Akadémia utca 6.","+3640484848",
-"MAVOHUHB","Sberbank Magyarország Zrt.","MAVOHUHB","base.hu",1088,"Budapest","Rákóczi  út 7.","info@sberbank.hu","+3614114200"
```
Comment (lordkrandel) [inline addons/l10n_hu/data/res.bank.csv:20]: Wait a minute... where did Sberbank go? 😮 
Not available anymore in Hungary? Sanctions?

**Comment by lordkrandel** [comment]
Since partner has vanished, a new PR odoo/odoo#181547 has been created in place of this.

**Pull request closed without merging**